### PR TITLE
Just some broken links and simple formatting...

### PR DESCRIPTION
**Pull request by al-the-x**
Also an excuse to open a PR for discussion. :D

### CODE REVIEW

File: js-functions-datatypes-math-1/README.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ After completing this assignment, you should be able to effectively use:
 
 ## Easy Mode
 
-Create a new Github repo and add [./exercises-1.js] to it. Make your edits to the functions in the "exercises-1" file.
+Create a new Github repo and add [`exercises-1.js`](exercises-1.js) to it. Make your edits to the functions in that file.
```
Comment (al-the-x) [inline js-functions-datatypes-math-1/README.md]: In the **Requirements** section, you mention a repo containing `main.js` but these instructions refer to `exercises-1.js`...

File: js-functions-practice/README.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ After completing this assignment, you should be able to effectively use:
 
 ## Normal Mode
 
-Create a new Github repo and add [./exercises-1.js] to it. Make your edits to the functions in the "exercises-1" file.
+Create a new Github repo and add [`exercises-1.js`](exercises-1.js) to it. Make your edits to the functions in that file.
```
Comment (al-the-x) [inline js-functions-practice/README.md]: Same here as above...

**Comment by al-the-x** [comment]
@matthiasak In `js-functions-practice/README.md` under **Performance Objectives**, you mention:

> Be able to write SOLID code that is clean, readable, and testable

But that seems well out of scope for the assignment... Is that just boilerplate from somewhere?

**Comment by matthiasak** [comment]
@al-the-x Correctomundo!

**Comment by matthiasak** [comment]
+1 :)

**Comment by al-the-x** [comment]
On a related note, I've been trying to keep up with the **Objectives** for my assignments this time around, and it's been burdensome enough that I've punted them for the time being. I have objectives in mind, but distilling them into bullets and sentences is Hard :tm: 

Planning to come back through on a second pass to fill those in.

**Comment by al-the-x** [comment]
@matthiasak To avoid boilerplate like that or burning time pontificating on _real_ objectives, what do you think about leaving that section as **TBD**...?

**Comment by matthiasak** [comment]
Potentially. Especially with HTML/CSS stuff it can sometimes be difficult. 

I'd rather keep some of the SOLID mentions in various assignments because it does encourage us to talk about it; and if we forget, students will hopefully ask.

Writing clean code is something I'm going to encourage daily; and this is part of those reminders.

Thoughts?

**Comment by matthiasak** [comment]
@al-the-x

**Comment by al-the-x** [comment]
I would agree with you about putting flags in the sand to remind ourselves of the topics that we _should_ cover in class. I've found it helpful to have a checklist of concepts that I drop into an Issue / PR for the day that is a part of the assignment. Topics that I want to cover for the day or parts of the previous or upcoming assignment that I want to build with them make recurring appearances.

**Pull request merged**